docs/src/regression-testing.md

```diff
@@ -56,6 +56,7 @@ In particular, the Kani testing suites are composed of:
                 Arguments can be specified in the `Cargo.toml` configuration file.
                 Similar to the `expected` suite, we look for `*.expected` files
                 for each harness in the package.
+* `cargo-ui`: Similar to `cargo-kani`, but focuses on the user interface like the `ui` test suite.
```

Comment:
Every additional suite we add loses us parallelism in the test suite. Is this really necessary?

Reply:
I think it's cleaner to have suites that test specific things. I don't think the impact on the time the regression runs is significant.